pkg/models/manifest.go

```diff
@@ -153,6 +158,20 @@ func BootstrapManifests(ctx context.Context, fluxClient flux.Flux, gitProvider g
 	}, nil
 }
 
+func WegoImageAndVersion() (string, string) {
+	version := version.Version
+	if os.Getenv("IS_TEST_ENV") != "" {
+		version = "latest"
+	}
+
+	image := WegoImage
+	if os.Getenv("IS_LOCAL_REGISTRY") != "" {
+		image = localWegoImage
+	}
+
+	return image, version
```

Comment:
I was going to combine so both image&version are configured through `IS_TEST_ENV`, but I wasn't sure if some folks still want to be able to deploy the `latest` `gchr` image using this environment variable